contracts/protocols/mainnet/augmented-finance/v1/main.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity ^0.8.4;
+
+import "./interfaces.sol";
+import "./helpers.sol";
+
+contract Resolver is Helpers {
+    /**
+     * @dev Get positions on Augmented Finance
+     * @param user user address
+     * @param tokens token addresses
+     */
+    function getPositions(address user, address[] memory tokens)
+        public
+        view
+        returns (UserTokenData[] memory, UserData memory)
+    {
+        IMarketAccessController mac = IMarketAccessController(getMarketAccessController());
```

Comment:
why not to make getMarketAccessController to be typed as IMarketAccessController to avoid casting here?

Reply:
It would be bettter if I rename it to `getMarketAccessControllerAddress` because `getMarketAccessController` returns address.